### ISSUE TITLE
Keyboard accessibility , cannot tabulate the component ngx-int-tel-input

### ISSUE THREAD

**Issue by bouyaahmedsami** [webcat12345/ngx-intl-tel-input#471]
I'm trying to use ngx-int-tel-input for my project so I referred to the demo page of component . They said that it is keyboard accessible, however, we cannot tabulate the component, and have access to it with keyboard arrows . Do you have any backlog item to make it compliance with accessibility in near future ? Do you have any tips , or solution to have the component accessible with keyboard ?

Thank you

**Comment by azeouinene** [comment]
I encounter the same problem, the dropdown is not accessible via keyboard , we can only access to the input.
in my case i want let the choice to the user for selecting the country (and dail code), but it doesn't focus on the element when clicking the tab button. Do you have any idea how can i access and open the dropdown (country/dial code) with keyboard ?
i tried the tabIndex as solution, but can't execute the event click on the dropdown because it's encapsulated in a div.

Thanks for help.

**Comment by cmckni3** [comment]
Duplicate of #421
